lib/cocoapods/target/framework_paths.rb

```diff
@@ -25,7 +30,9 @@ def ==(other)
       alias eql? ==
 
       def hash
-        if (dsym = dsym_path)
+        if (dsym = dsym_path && bcsymbolmaps = bcsymbolmap_paths)
```

Comment:
do we miss another case here of:

```ruby
...
elseif (bcsymbolmaps = bcsymbolmap_paths
  [source_path, bcsymbolmaps].hash
...
```

Reply:
also I think this can be simplified with:
```ruby
[source_path, dsym, bcsymbolmaps].hash
```
Nil works for `hash`.
```
2.3.1 :002 > nil.hash
 => 962373697272586016
```

cc @amorde

Reply:
yes that will work, `Array` handles `nil` instances in its implementation of `hash` AFAIK